modules/joda/src/test/scala/pureconfig/module/joda/JodaSpec.scala

```diff
@@ -0,0 +1,62 @@
+package pureconfig.module.joda
+
+import com.typesafe.config.{ ConfigFactory, ConfigValueFactory }
+import org.joda.time._
+import org.scalacheck.Arbitrary
+import JodaSpec._
+import org.joda.time.format.{ DateTimeFormat, DateTimeFormatter }
+import pureconfig.module.joda.configurable.ConfigurableSuite.{ dateTimeArbitrary, zoneIdArbitrary }
+import pureconfig.BaseSuite
+import pureconfig.syntax._
+
+class JodaSpec extends BaseSuite {
```

Comment:
@queimadus Terribly sorry. I remembered the convention in PureConfig incorrectly. This should be renamed to `JodaSuite`. That's entirely my fault.